js/actions/appActions.js

```diff
@@ -1337,6 +1337,21 @@ const appActions = {
       hostPattern: origin,
       key: 'savePasswords'
     })
+  },
+
+  /**
+   * Indicates that the urlbar text has changed, usually from user input
+   *
+   * @param {string} location - The text to set as the new navbar URL input
```

Comment:
derp- this is an individual commit, will go back to looking at the all files changed view 😛  Disregard